embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/injection/ConfigModuleImpl.kt

```diff
@@ -0,0 +1,50 @@
+package io.embrace.android.embracesdk.internal.injection
+
+import io.embrace.android.embracesdk.internal.Systrace
+import io.embrace.android.embracesdk.internal.config.ConfigService
+import io.embrace.android.embracesdk.internal.config.EmbraceConfigService
+import io.embrace.android.embracesdk.internal.config.LocalConfigParser
+import io.embrace.android.embracesdk.internal.payload.AppFramework
+import io.embrace.android.embracesdk.internal.worker.WorkerName
+
+internal class ConfigModuleImpl(
+    initModule: InitModule,
+    coreModule: CoreModule,
+    openTelemetryModule: OpenTelemetryModule,
+    workerThreadModule: WorkerThreadModule,
+    androidServicesModule: AndroidServicesModule,
+    customAppId: String?,
```

Comment:
I think we should seriously think about getting rid of this in 7.0

Reply:
Agree